src/custom-types.json

```diff
@@ -455,6 +455,12 @@
   "audio/basic": {
     "compressible": false
   },
+  "audio/flac": {
+    "sources": [
+      "https://packages.debian.org/buster/mime-support"
```

Comment:
This needs to be a primary source; links to type aggregating sites and Wikipedia are not acceptable.

Reply:
Debian package is not a primary source? This is *the* MIME database for Debian distributions, including Ubuntu.

Reply:
Sure, I understand that, but they are not the authority on the FLAC mime type, are they? The authority on the type would be the primary source. I assume they sourced this mapping from somewhere, so that would be what to like here; they are by nature (same at this module) not a primary source of this mapping data.

Reply:
Xiph.Org Foundation seems to be the owners of the FLAC format/specs and it looks like this is the primary source you are looking to link to: https://wiki.xiph.org/index.php?title=MIME_Types_and_File_Extensions

Reply:
@sgpinkus I think you may have resolved this on accident, as the link doesn't seem to be replaced still.